runtime.go

```diff
@@ -334,13 +334,22 @@ func (e *Exception) NativeError() error {
 }
 
 func (e *Exception) StringifyError(r *Runtime) string {
-	val := r.builtinJSON_stringify(FunctionCall{
-		Arguments: []Value{
-			e.val,
-		},
-	})
+	// if the exception has a base type of Object or Array then stringify the result
+	base := e.val.baseObject(r)
+	if base != nil {
+		switch base.ClassName() {
+		case "Array", "Object":
```

Comment:
Reworked this function a bit to only go through json stringify for types that we will have a harder time with in baas. These should also be exceptional (maybe?) if a user were to do something like this:
```
throw { foo: "bar" };
// or
throw [1, 2, 3];
```

Not really sure why they would do that but they could and this should stringify so that we can see that value as the "error" in baas